Userland/Utilities/passwd.cpp

```diff
@@ -111,10 +112,12 @@ int main(int argc, char** argv)
 
         if (new_password.value() != new_password_retype.value()) {
             warnln("Sorry, passwords don't match.");
+            warnln("Password for user {} unchanged.", target_account.username());
             return 1;
         }
 
         target_account.set_password(new_password.value().characters());
+        outln("Password for user {} successfully updated.", target_account.username());
```

Comment:
This isn't true yet. It wasn't updated until after the call to `sync()` below succeeds.

Reply:
Looks great other than this one thing :)

Reply:
Made the change 👍